benchmark/buffers/dataview-set.js

```diff
@@ -41,17 +41,19 @@ function main(conf) {
 
 function benchInt(dv, fn, len, le) {
   var m = mod[fn];
+  var method = dv[fn];
```

Comment:
If it affects the numbers (I know it can when it comes to `let`), feel free to leave it, but it’s become pretty standard around here to always use `const` when it makes sense… personally, I don’t care much, though. :smile: